static/app/views/issueDetails/allEventsTable.tsx

```diff
@@ -24,15 +25,35 @@ export interface Props {
   excludedTags?: string[];
 }
 
+const makeGroupPreviewRequestUrl = ({groupId}: {groupId: string}) => {
+  return `/issues/${groupId}/events/latest/`;
+};
+
 function AllEventsTable(props: Props) {
   const {location, organization, issueId, excludedTags, group} = props;
   const config = getConfigForIssueType(props.group);
   const [error, setError] = useState<string>('');
   const routes = useRoutes();
   const {fields, columnTitles} = getColumns(group, organization);
 
+  const endpointUrl = makeGroupPreviewRequestUrl({
+    groupId: group.id,
+  });
+
+  const {data} = useApiQuery<EventTransaction>([endpointUrl, {query: {}}], {
```

Comment:
nit: you should just be able to pass `[endpointUrl]` here if you don't need a query string

Reply:
👍